app/app.go

```diff
@@ -8,14 +8,14 @@ import "fmt"
 // TODO: What does the program do?
 func main() {
 	var num uint
-	fmt.Scan(&num)
-
+	fmt.Scan(&num) // person enters number
+	// this program is similar to program in homework wich places numbers in order
```

Comment:
It's a great observation, Anton! :) Well done with recognising the same patterns in different programs.